release-calendar/src/server/repository-service.ts

```diff
@@ -57,4 +60,43 @@ export class RepositoryService {
 
     return releaseQuery;
   }
+
+  async getCurrentReleases(): Promise<Promotion[]> {
+    const rollbackQuery = this.promotionRepository
+      .createQueryBuilder('promotion')
+      .where('promotion.channel = :channel', {channel: Channel.ROLLBACK})
+      .select('promotion.releaseName')
+      .groupBy('promotion.releaseName')
```

Comment:
You won't need to group by release name to get rollbacked releases.

Reply:
I couldn't find good documentation on this, but it was the only way I could figure out how to get distinct releaseNames since we're using mySql. https://typeorm.io/#/select-query-builder/adding-distinct-on-expression-postgres-only

Reply:
It should always be distinct because `release.name` is a PK